Library/Formula/subversion.rb

```diff
@@ -156,6 +151,8 @@ def install
     if MacOS::CLT.installed?
       args << "--with-apr=/usr"
       args << "--with-apr-util=/usr"
+      # Ignore authz API bug in system httpd (fixed in httpd 2.4.16)
+      args << "--enable-broken-httpd-auth" if !MacOS::CLT.installed? && MacOS.version < :el_capitan
```

Comment:
Should have not touch this yesterday, was too tired. This line is not needed with brew ```apr```, it compiles without issues. @brainy am I correct on this?